lexers/c/crontab.go

```diff
@@ -0,0 +1,18 @@
+package c
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// Crontab lexer.
+var Crontab = internal.Register(MustNewLexer(
```

Comment:
Is this a new lexer?

Reply:
This is a wakatime top language: https://wakatime.com/leaders/languages.